controls/cis_ocp_1_4_0/section-1.yml

```diff
@@ -232,70 +255,89 @@ controls:
+      status: automated
+      rules:
+        - api_server_audit_log_path
+        - openshift_api_server_audit_log_path
       level: level_1
     - id: 1.2.21
       title: Ensure that the audit logs are forwarded off the cluster for retention
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - audit_log_forwarding_enabled
+        - audit_log_forwarding_webhook
       level: level_1
     - id: 1.2.22
       title: Ensure that the maximumRetainedFiles argument is set to 10 or as appropriate
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_audit_log_maxbackup
+        - ocp_api_server_audit_log_maxbackup
       level: level_1
     - id: 1.2.23
       title: Ensure that the maximumFileSizeMegabytes argument is set to 100
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_audit_log_maxsize
+        - ocp_api_server_audit_log_maxsize
       level: level_1
     - id: 1.2.24
       title: Ensure that the --request-timeout argument is set
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_request_timeout
       level: level_1
     - id: 1.2.25
       title: Ensure that the --service-account-lookup argument is set to true
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_service_account_lookup
       level: level_1
     - id: 1.2.26
       title: Ensure that the --service-account-key-file argument is set as appropriate
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_service_account_public_key
       level: level_1
     - id: 1.2.27
       title: Ensure that the --etcd-certfile and --etcd-keyfile arguments are set
         as appropriate
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_etcd_cert
+        - api_server_etcd_key
       level: level_1
     - id: 1.2.28
       title: Ensure that the --tls-cert-file and --tls-private-key-file arguments
         are set as appropriate
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_tls_cert
+        - api_server_tls_private_key
       level: level_1
     - id: 1.2.29
       title: Ensure that the --client-ca-file argument is set as appropriate
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_client_ca
       level: level_1
     - id: 1.2.30
       title: Ensure that the --etcd-cafile argument is set as appropriate
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_etcd_ca
       level: level_1
     - id: 1.2.31
       title: Ensure that encryption providers are appropriately configured
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - api_server_encryption_provider_cipher
```

Comment:
This one says `    cis@ocp4: 1.2.31,1.2.32,2.8`

Reply:
For `1.2.31` there is this PR, which may actually be more accurate:
https://github.com/ComplianceAsCode/content/pull/10753